compiler/riscv/codegen/OMRTreeEvaluator.cpp

```diff
@@ -399,6 +399,22 @@ OMR::RV::TreeEvaluator::directCallEvaluator(TR::Node *node, TR::CodeGenerator *c
    return linkage->buildDirectDispatch(node);
    }
 
+// handles calli, icalli, lcalli, fcalli, dcalli, acalli
+TR::Register *
+OMR::RV::TreeEvaluator::indirectCallEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR::SymbolReference *symRef = node->getSymbolReference();
+   TR::MethodSymbol *callee = symRef->getSymbol()->castToMethodSymbol();
+
+   // FIXME: How comes here we get private linkage?
+   // TR::Linkage *linkage = cg->getLinkage(callee->getLinkageConvention());
```

Comment:
You reported this problem to me before.  `TR_Private` is the default linkage set when you create a `MethodSymbol`.   I started the work to make `TR_System` the default linkage (which it should be in OMR) but I ran into problems in OpenJ9 and initializing the linkage correctly at the right moment on the symbol.  I admit this fell off my radar and I don't recall all the issues I faced, but I will create an issue in OpenJ9 to track the investigation and work to make this real.

Reply:
FYI: eclipse/openj9#9741